app/src/main/res/layout/activity_sign_in.xml

```diff
@@ -6,16 +6,75 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:padding="20dp"
+    android:background="@drawable/bg_gradient"
     tools:context=".SignIn">
 
+
+    <TextView
+        android:id="@+id/textView2"
```

Comment:
Perhaps give this a better id that better explains what this text view is used for.